### DISCUSSION TITLE
Out of memory allocating 301989888 bytes!

### DISCUSSION THREAD

**Discussion by Manju9876** [learndevopsonline/learndevopsonline#654] [Q&A] [answered]
hello @r-devops sir,
when  I reran the shipping.sh file every thing goes fine but while in the installation part,
 the script prompts like this message.

###Out of memory allocating 301989888 bytes!

![Screenshot 2023-04-21 192847](https://user-images.githubusercontent.com/129134583/233655894-d22d2a81-fc30-483e-a03a-5b38f491ba5c.png)

**Comment by noolivara** [comment, marked as answer]
@Manju9876 Go with t3.small instance instead of t3.micro.

**Reply by Manju9876** [reply]
okay sir, got it
